frontend/src/components/Secret.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState, useEffect } from 'react'
+import { useDispatch, useSelector } from "react-redux"
+import { auth } from "../reducers/auth"
+import { BeatLoader } from 'react-spinners'
+import styled from 'styled-components/macro'
+import moment from 'moment'
+
+const Main = styled.main`
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    height: 100vh;
+    background: #fff;
+`
+
+const Loading = styled.main`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+`
+
+const SecretForm = styled.form`
+    display: flex;
+    justify-content: flex-start;
+    width: 50vw;
+`
+
+const SecretText = styled.input`
+    border-style: hidden;
+    border-radius: 5px;
+    border-bottom: 2px solid black;
+    background-color: #eee;
+    width: 80%;
+    height: 30px;
+    font-size: 20px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    padding: 0px 10px;
+    @media (min-width: 768px) {
+        width: 40vw;
+    }
+`
+
+const AddSecret = styled.input`
+    border: none;
+    font-size: 20px;
+    line-height: 20px;
+    margin: 0px 10px;
+    padding: 0px;
+    text-align: center;
+    opacity: ${props => (props.active ? "1" : "0.6")};
+`
+
+const SecretList = styled.ul`
+    list-style: none;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 50vw;
+`
+
+const StyledSecret = styled.li`
+    background: #fff;
+`
+
+const SecretDate = styled.p`
+    font-size: 12px;
+`
+
+export const Secret = () => {
+    const [secrets, setSecrets] = useState([])
+    const [sentMessage, setSentMessage] = useState(false)
+    const [status, setStatus] = useState()
+    const [loading, setLoading] = useState(true)
+    const [message, setMessage] = useState('')
+    const dispatch = useDispatch()
+    const token = useSelector(store => store.auth.accessToken)
+    const userId = useSelector(store => store.auth.userId)
+    const name = useSelector(store => store.auth.name)
+
+    const logout = () => {
+        dispatch(auth.actions.logout())
+    }
+
+    const sendSecret = (event) => {
+        event.preventDefault()
+
+        if (message !== "") {
+            fetch(`https://authentication-tiago-ivett.herokuapp.com/secret/${userId}`, {
+                method: "POST",
+                body: JSON.stringify({ message }),
+                headers: { "Content-Type": "application/json", "Authorization": token }
+            }).catch(err => console.log("error:", err))
+        }
+        setMessage("")
+        setSentMessage(!sentMessage)
+    }
+
+    useEffect(() => {
+        if (userId) {
+            fetch(`http://localhost:8080/secrets/${userId}`, {
+                headers: {
+                    "Authorization": token
+                }
+            })
```

Comment:
Oops! `localhost`
This is where using `const BASE_URI = https://authentication-tiago-ivett.herokuapp.com/` is super useful!  Much harder to miss!

Reply:
This is the cause of why your secrets weren't showing up.  I got it to work locally! Looks cooool 💯